modules/plottinggl/glsl/pcp_axis.vert

```diff
@@ -2,6 +2,11 @@
 
 uniform float x;
 
-void main(){
-    gl_Position = vec4(getPosWithSpacing(vec2(x,in_Vertex.y)),0,1);
-} 
\ No newline at end of file
+out vec3 pickColor;
+out vec2 texCoord;
+
+void main() {
+    pickColor = in_Normal.rgb;
```

Comment:
Normal?

Reply:
I don't know. Copied it from the shader for the PCP lines.

Reply:
You are setting the pickColor uniform in cpp code. Should not use the normal